js/src/components/paid-ads/campaign-preview/components/scaled-text.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
+const styleName = {
+	adBadge: 'gla-ads-mockup__scaled-text--ad-badge',
+	smaller: 'gla-ads-mockup__scaled-text--smaller',
+	larger: 'gla-ads-mockup__scaled-text--larger',
+	blue: 'gla-ads-mockup__scaled-text--blue',
+	'gray-700': 'gla-ads-mockup__scaled-text--gray-700',
+	'gray-800': 'gla-ads-mockup__scaled-text--gray-800',
+};
+
+/**
+ * Renders a placeholder to draw a line.
+ *
+ * @param {Object} props React props.
+ * @param {boolean} [props.adBadge=false] Whether to show the 'AD' badge.
+ * @param {boolean} [props.smaller] Whether to draw a smaller text size.
+ * @param {boolean} [props.larger] Whether to draw a larger text size.
```

Comment:
💅  Same as the case of thinner/ticker... Why not just set a type prop and add the class name modifier? Or do the inline style mdoification?

Reply:
Changed in https://github.com/woocommerce/google-listings-and-ads/pull/1650/commits/631e8fe5f5f680ea82ee2c427a6d51617817669a.